.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+.pylintrc
```

Comment:
No need to ignore this file

Reply:
this is file with settings for Pylint extension for Visual Studio Code i added it in project myself

Reply:
i do not want to remove it from my project but it wasn't in forked files